tests/components/advantage_air/test_init.py

```diff
@@ -20,6 +24,10 @@ async def test_async_setup_entry(hass, aioclient_mock):
     entry = await add_mock_config(hass)
     assert entry.state == ENTRY_STATE_LOADED
 
+    assert await hass.config_entries.async_unload(entry.entry_id)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)